stories/topics/con-aws-get-deployment-info.adoc

```diff
@@ -0,0 +1,8 @@
+[id="con-aws-get-deployment-info"]
+
+= Retrieve deployment information
+
+After deploying {PlatformNameShort}, follow the procedures below to retrieve information about your deployment.
+
+include::../topics/proc-aws-retrieve-admin-password.adoc[leveloffset=+1]
```

Comment:
con-get-deployment-info.adoc is already in /topics/ so if you include that, everything included in that is already at that level, hence

include::proc-aws-retrieve-admin-password.adoc[leveloffset=+1]